### DISCUSSION TITLE
How access many rows in XML format ?

### DISCUSSION THREAD

**Discussion by GiovanniSinosini** [grafana/grafana-infinity-datasource#172] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/56783045/137174471-ee102553-b977-4db7-ab43-a6475a039a67.png)

In Rows / Root, returns only one result in this format. I need it to return all Request results, not just one line. For example:
I have that => feed.entry[0]   = work
I need that => feed.entry[all] => but it´s not work. 

Please, Can someone help me?

**Comment by yesoreyeram** [comment]
Can you post the sample data?

**Reply by GiovanniSinosini** [reply]
![image](https://user-images.githubusercontent.com/56783045/137193955-f92830db-afe0-43cc-a9c0-2a75fb0d1810.png)

...
many others "entry" components.
I need to count all status components. Thank you.

**Comment by yesoreyeram** [comment, marked as answer]
Minimal way to reproduce this, 

```xml
<feed>
	<id></id>
	<title></title>
	<entry>
		<title>hello world</title>
		<content>
			<m:properties>
				<d:logStart>2021-09-24T23:12:30.123</d:logStart>
				<d:status>OK</d:status>
			</m:properties>
		</content>
	</entry>
	<entry>
		<title>hello mars</title>
		<content>
			<m:properties>
				<d:logStart>2021-09-25T22:10:30.123</d:logStart>
				<d:status>Not OK</d:status>
			</m:properties>
		</content>
	</entry>
</feed>
```

Use the following settings. 

Root : `feed.entry`
Columns : 
| name | selector | format |
|---|--|--|
| title | `title` | string |
| status | `content[0]["m:properties"][0]["d:status"]` | string | 
| log start | `content[0]["m:properties"][0]["d:logStart"]` | timestamp |

![image](https://user-images.githubusercontent.com/153843/137256481-18da04dd-e44e-4367-a986-431d77e6832c.png)


PS: Mark it **answered** if helps.

**Reply by GiovanniSinosini** [reply]
Awesome. In the documentation, this subject is little explored. Thank you very much.
